server/src/index.ts

```diff
@@ -7,7 +7,7 @@ import scraper from "../utils/scraper";
 import { setupDb } from "./database/setup";
 import baseRouter from "./routes/index";
 
-const log = parentLogger.child({ module: "express" });
+const log = parentLogger.child({ module: "router" });
```

Comment:
This just reminded me of something: I've been noticing that we have `server    |     module: "express"` gets printed on every other line, is that intended?

Reply:
when i set up the logger, i ended up changing that line from `"express"` to `"router"` (as per rob's suggestion in #46), but i think maybe another branch set it to `"express"` after? `server | module: "express"` definitely wasn't intended behaviour on my part, but this should fix it

Reply:
I changed that to express because we actually have routes now, so I added module: "router" to our rest api.

Reply:
oh sorry, i might've missed that in your pr! since we have `module: router/coursesection` and `module: router/prereq` in the controller functions, i think maybe this should be `module: router` instead as it's still kinda part of the base router? i might be looking in the wrong place, but i don't see a `module: router` anywhere (except for the db). 

very minor thing though lol